README.md

```diff
@@ -30,27 +30,55 @@ jobs:
 
 For more usage examples please look for the `_test-*` workflow definition in the [`.github/workflows/`](.github/workflows/) folder.
 
+## Quick Start Guide for Reusable Workflows
+
+To get started with using reusable workflows in your repository, follow these steps:
+
+1. Create a `.github/workflows` directory in your repository if it doesn't already exist.
+
+2. Copy an example workflow file from the reusable workflows repository into your `.github/workflows` directory. You can find the workflow example files in the [`.github/workflows/samples`](.github/workflows/samples) folder of this repository.
+
+3. Open the workflow file you just copied and customize it according to your needs. Each workflow file contains a set of jobs that define the tasks to be executed.
+
+4. Make sure to set the required inputs for the workflow. These inputs are defined in the workflow file and can be customized to match your specific requirements. For example, you might need to provide values for environment variables, AWS credentials, or Docker image details.
+
+5. Save the modified workflow file.
+
+6. Commit and push the changes to your repository.
+
+7. The workflow will now be triggered based on the specified events. For example, if you have configured the workflow to run on every push event, it will automatically start whenever you push changes to your repository.
+
+That's it! You have successfully set up and customized a reusable workflow in your repository. Now you can benefit from the predefined tasks and automation provided by the workflow to streamline your development process.
+
+For the list and documentation of the maintained workflows, please have a look below!
+
 ## Maintained workflows
 
 Each workflow has its own dedicated documentation. For a detailed understanding, you should go through the code and understand each step and how they interact with each other.
 
-## Terraform Deployment
+Most of the workflows do some "black magic", by getting values by default from your Github Environments, if set as `environment` input.
+
+### Terraform Deployment
+
+This workflow is responsible for deploying infrastructure using Terraform through dflook. For more information see [Terraform Deployment Documentation](/docs/README-tf-deploy.md).
+
+#### Feature branches deployment
```

Comment:
Small suggestion: move the documentation to the Wiki of this repo. This way, there is no need of a PR to do any updates to the documentation.

Reply:
I think reviewing of docs is important :) 

Since this is a public repo, I have a proposal for moving to mkdocs and github pages which can be referenced and linked much easier.